Main.js

```diff
@@ -0,0 +1,71 @@
+
+let renderer = new Renderer()
+let board = new GoldRush(5, 5)
+renderer.renderBoard(board.matrix, 5, 5, board.playerScores)
+
+
+$(document).keypress(function (e) {
```

Comment:
Use one event listener to handle all moves. Inside you can use a bunch of `if` statements.

You can also use a `switch-case` statement or an object mapping the key to the correct action.